src/ProjectNeon/Assets/Features/Effects/EffectOnAttacker.cs

```diff
@@ -20,15 +20,24 @@ void Effect.Apply(Member source, Target target)
     {
         _performer = source;
         _effectTarget = target;
-        BattleEvent.Subscribe<Attack>(attack => Execute(attack), this);
+        BattleEvent.Subscribe<AttackPerformed>(attackPerformed => Execute(attackPerformed.Attack), this);
     }
 
     void Execute(Attack attack)
     {
         _effectTarget.Members.ForEach(
             target => {
-                if (target.Equals(attack.Target))
+                /**
+                 * @todo #454:30min Refactor Target - Member relation. At the moment the responsible for discovering target
```

Comment:
This puzzle is not the work we want. No language about refactoring. 

We only care that we never index into `Target.Members`.

Should be 15 mins.

Reply:
@SilasReinagel Done